docs/core/tools/dotnet-sln.md

```diff
@@ -131,98 +131,80 @@ dotnet sln [<SOLUTION_FILE>] remove <PROJECT_PATH> [<PROJECT_PATH>...]
 dotnet sln [<SOLUTION_FILE>] remove [-h|--help]
 ```
 
-#### Arguments
-
-- **`SOLUTION_FILE`**
+#### Description
+The `dotnet sln remove` command removes one or more projects from a `.sln` (solution) file.  
+This command updates the solution file but **does not delete** the project files from disk.
 
-  The solution file to use. If it is unspecified, the command searches the current directory for one and fails if there are multiple solution files.
+#### Arguments
 
-- **`PROJECT_PATH`**
+- **`SOLUTION_FILE`**  
+  The solution file to modify. If unspecified, the command searches for a `.sln` file in the current directory.  
```

Comment:
As of 9.0.200, this isn't strictly true anymore - slnx files will be considered as well if they are present in the current directory.